lib/srv/desktop/discovery.go

```diff
@@ -48,11 +49,15 @@ const (
 	// gmsaClass is the object class for group managed service accounts in Active Directory.
 	gmsaClass = "msDS-GroupManagedServiceAccount"
 
-	attrName        = "name"
-	attrDNSHostName = "dNSHostName" // unusual capitalization is correct
-	attrObjectGUID  = "objectGUID"
-	attrOS          = "operatingSystem"
-	attrOSVersion   = "operatingSystemVersion"
+	writableDomainControllerGroupID = "516"
+	readOnlyDomainControllerGroupID = "521"
```

Comment:
Nit: would probably be helpful to add a link to the official documentation where these IDs are defined (if there's any).